merge-v1/src/mapping.ts

```diff
@@ -55,7 +69,7 @@ function checkMergeClass(_mergeClass: BigInt): String {
  * NOTE: Checks through different scenarios (current WIP covers): 1. minting
  * TODO: 2. transfer from NG to ext. addresses, 3. transfers btw ext. addresses to other ext. addresses (not whitelisted), 4. merge() being called, 5. possibly merge being called by NG specifically.
  */
-export function handleTransfer(event: Transfer): void {
```

Comment:
do not remove export - this makes it so that handleTransfer is not accessible to the subgraph, and thats why you are getting the error